automation/run-integration-tests.sh

```diff
@@ -105,7 +105,30 @@ function open_shell {
     run_exit
 }
 
-cd $EXEC_PATH 
+options=$(/usr/bin/getopt --options "" --long pytest-args:,help,debug-shell -- "${@}")
```

Comment:
Do we have to specify the full path? On different distributions it may recide on different paths.

Reply:
good catch, fixed